integration_tests/src/main/python/delta_lake_write_test.py

```diff
@@ -1009,7 +1009,8 @@ def test_delta_write_optimized_supported_types_partitioned(spark_tmp_path):
 @delta_lake
 @ignore_order(local=True)
 @pytest.mark.skipif(is_before_spark_320(), reason="Delta Lake writes are not supported before Spark 3.2.x")
-@pytest.mark.skipif(not is_databricks_runtime(), reason="Delta Lake optimized writes are only supported on Databricks")
+@pytest.mark.skipif(not is_databricks_runtime() and is_before_spark_353(), reason="Delta Lake optimized writes are only supported on Databricks")
+@pytest.mark.xfail(is_spark_353_or_later(), reason="We support Optimized write on GPU for Spark 3.5.3+")
```

Comment:
nit: I understand the need to improve readability but is now the real intent of the PR (this one line) is buried in many more unrelated changed lines? Should the PR title then say "Replace skip with xfail for optimized write  for test_delta_write_optimized_unsupported_sort_fallback"?

Reply:
It is more clear now without xfail. thanks